system/src/system_power_manager.cpp

```diff
@@ -377,7 +378,7 @@ void PowerManager::loop(void* arg) {
     hal_gpio_mode(PMIC_INT, INPUT_PULLUP);
 #if HAL_PLATFORM_SHARED_INTERRUPT
     hal_interrupt_extra_configuration_t extra = {};
-    extra.version = HAL_INTERRUPT_EXTRA_CONFIGURATION_VERSION_1;
+    extra.version = HAL_INTERRUPT_EXTRA_CONFIGURATION_VERSION_3;
```

Comment:
`HAL_INTERRUPT_EXTRA_CONFIGURATION_VERSION`

Reply:
As discussed, the philosophy is to use the latest version/size, make sure all fields are zero-inited, and ensure that the HAL that access the struct handles zero inited fields appropriately